apiserver/facades/client/bundle/bundle.go

```diff
@@ -91,3 +99,78 @@ func (b *bundleAPI) GetChanges(args params.BundleChangesParams) (params.BundleCh
 	}
 	return results, nil
 }
+
+// ExportBundle exports the model configuration specified in the argument as bundle.
+func (b bundleAPI) ExportBundle(args params.ExportBundleParams) (params.BundleData, error) {
```

Comment:
The model to export comes from the facade initialisation.
The NewFacade() method which currently ignores State needs to be changed.
The New method should also be converted to use a facade context.